package.json

```diff
@@ -42,7 +42,7 @@
     "vl-ui-core": "^7.1.3",
     "vl-ui-form-message": "^5.0.7",
     "vl-ui-search": "^1.1.9",
-    "vl-ui-select": "^4.2.1",
+    "vl-ui-select": "git+https://github.com/milieuinfo/webcomponent-vl-ui-select.git#UIG-1186-select-search-placeholders",
```

Comment:
Ik denk wel niet dat Tom dit gaat kunnen mergen naar master zonder dat hier een release nummer staat?

Reply:
ja, maar dat moet nog aangepast worden eenmaal er een release is van vl-ui-select. Ik had dat nodig om het geheel te kunnen testen.

Reply:
Ok, dat zal @coemans dan wel zien voor het mergen hoop ik :)

Reply:
Ik heb conversation unresolved zodat ik zeker nog reminder heb.